src/app/App.js

```diff
@@ -1,11 +1,11 @@
 define([
+    './config',
```

Comment:
Any particular reason to switch to relative imports? I've had problems in the past but it's been a long time and I don't remember the specifics.